php7.0/Dockerfile

```diff
@@ -49,6 +51,9 @@ RUN curl \
 RUN mkdir -p /app && rm -fr /var/www/html && ln -s /app /var/www/html
 WORKDIR /app
 
+# Add Composer's vendor/bin/ to the PATH
+ENV PATH "$PATH:/app/vendor/bin"
```

Comment:
Did you use `export PATH="$PATH:/app/vendor/bin` in `run.sh`? `export` being the important part.